### PR TITLE
Fix TAB alignment issue

### PR DESCRIPTION
**Pull request by mike-spa**
Resolves: #16088

### CODE REVIEW

**Review by oktophonie — APPROVED** [review]

File: src/engraving/layout/layoutchords.cpp [hunk 1]
```diff
@@ -478,27 +479,29 @@ void LayoutChords::layoutChords1(Score* score, Segment* segment, staff_idx_t sta
...
                         }
                     } else {
-                        chord->movePosX(centerUp);
-                    }
-                } else {
-                    if (downOffset != 0.0) {
-                        chord->movePosX(downOffset + centerAdjustDown);
-                        if (upDots && !downDots) {
-                            chord->movePosX(dotAdjust);
+                        if (downOffset != 0.0) {
```
Comment (RomanPudashkin) [inline src/engraving/layout/layoutchords.cpp:497]: Please use RealIsNull

File: src/engraving/layout/layoutchords.cpp [hunk 2]
```diff
@@ -478,27 +479,29 @@ void LayoutChords::layoutChords1(Score* score, Segment* segment, staff_idx_t sta
...
-                        chord->movePosX(upOffset + centerAdjustUp + oversizeUp);
-                        if (downDots && !upDots) {
-                            chord->movePosX(dotAdjust);
+        if (!isTab) {
+            for (track_idx_t track = partStartTrack; track < partEndTrack; ++track) {
+                EngravingItem* e = segment->element(track);
+                if (e && e->isChord() && toChord(e)->vStaffIdx() == staffIdx) {
+                    Chord* chord = toChord(e);
+                    if (chord->up()) {
+                        if (upOffset != 0.0) {
```
Comment (RomanPudashkin) [inline src/engraving/layout/layoutchords.cpp:488]: Please use RealIsNull

File: src/engraving/layout/layoutchords.cpp [hunk 3]
```diff
@@ -478,27 +479,29 @@ void LayoutChords::layoutChords1(Score* score, Segment* segment, staff_idx_t sta
...
-        for (track_idx_t track = partStartTrack; track < partEndTrack; ++track) {
-            EngravingItem* e = segment->element(track);
-            if (e && e->isChord() && toChord(e)->vStaffIdx() == staffIdx) {
-                Chord* chord = toChord(e);
-                if (chord->up()) {
-                    if (upOffset != 0.0) {
-                        chord->movePosX(upOffset + centerAdjustUp + oversizeUp);
-                        if (downDots && !upDots) {
-                            chord->movePosX(dotAdjust);
+        if (!isTab) {
```
Comment (RomanPudashkin) [inline src/engraving/layout/layoutchords.cpp:482]: Any new vtest for this case?

**Comment by mike-spa** [comment]
Done and added a dedicated vtest 👍

**Pull request merged**